aries-site/src/examples/templates/filtering/FilteringWithSelect.js

```diff
@@ -0,0 +1,368 @@
+            filterData(allData, previousFilters);
+            restoreValues(previousValues);
+            setShowLayer(!showLayer);
+          }}
+          onEsc={() => {
+            filterData(allData, previousFilters);
+            restoreValues(previousValues);
+            setShowLayer(!showLayer);
+          }}
+        >
+          <Box
+            width={{ min: 'medium' }}
+            pad={{ horizontal: 'medium', vertical: 'medium' }}
+            gap="medium"
+            fill="vertical"
+          >
+            <Header>
+              <Heading as="p" color="text-strong" margin="none">
+                Filters
+              </Heading>
+              <Button
+                icon={<FormClose />}
+                onClick={() => {
+                  filterData(allData, previousFilters);
+                  restoreValues(previousValues);
+                  setShowLayer(!showLayer);
+                }}
+              />
+            </Header>
+            <Box pad="xsmall" overflow="auto" flex>
+              {content}
+            </Box>
+            <Box align="center" direction="row" gap="small">
+              <Button
+                label="Apply Filters"
+                onClick={() => {
+                  const nextData = filterData(allData, filters);
+                  setData(nextData);
+                  setShowLayer(!showLayer);
+                }}
+                primary
+              />
+              <Button
+                label="Reset Filters"
+                onClick={() => {
+                  resetFilters();
+                  setShowLayer(!showLayer);
+                }}
+                secondary
+              />
+            </Box>
+          </Box>
+        </Layer>
+      )}
+    </>
+  );
+};
+
+Filters.propTypes = {
+  filters: PropTypes.shape({}).isRequired,
+  setFilters: PropTypes.func.isRequired,
+  filtering: PropTypes.bool.isRequired,
+  filterData: PropTypes.func.isRequired,
+  setData: PropTypes.func.isRequired,
+  setFiltering: PropTypes.func.isRequired,
+};
+
+const RecordSummary = ({ data, filtering }) => (
+  <Box direction="row" gap="xxsmall">
+    <Text size="small" weight="bold">
+      {data.length}
+    </Text>
+    <Text size="small">{filtering ? 'results of ' : 'items'}</Text>
+    {filtering && (
+      <Box direction="row" gap="xxsmall">
+        <Text size="small" weight="bold">
+          {allData.length}
+        </Text>
+        <Text size="small">items</Text>
+      </Box>
+    )}
+  </Box>
+);
+
+RecordSummary.propTypes = {
+  data: PropTypes.arrayOf(
+    PropTypes.shape({
+      availability: PropTypes.number,
+      name: PropTypes.string,
+      location: PropTypes.string,
+    }),
+  ).isRequired,
+  filtering: PropTypes.bool,
+};
+
+const Results = ({ data }) => (
+  <Box pad={{ bottom: 'medium' }} overflow="auto" fill>
+    <List
+      background="background-front"
+      border="horizontal"
+      data={data}
+      action={(item, index) => (
```

Comment:
Getting squished. 

Truncate? Responsive gap? Responsive font-size?
![Screen Shot 2020-09-10 at 4 38 46 PM](https://user-images.githubusercontent.com/1756948/92818078-86bd3180-f384-11ea-8cb6-420ed1d27d6f.png)

Reply:
Thinking maybe the "ready"/  "paused" text should my omitted on mobile